Apps/frontend/src/labels.js

```diff
@@ -1,12 +1,8 @@
 /* Labels used in the UI */
 
-export const LABEL_BUTTON_MANAGE_CHANNEL_CONFIG = "Manage presets";
 export const LABEL_HEADER_LOAD_FROM_LIST_CHANNEL_CONFIG =
-  "Load existing preset from list:";
-export const LABEL_HEADER_CREATE_CHANNEL_CONFIG = "Create new preset:";
-export const LABEL_BUTTON_LOAD_CHANNEL_CONFIG = "Load";
-export const LABEL_BUTTON_UPDATE_CHANNEL_CONFIG = "Save";
-export const LABEL_BUTTON_CANCEL_CHANNEL_CONFIG = "Cancel";
+  "Load Existing:";
```

Comment:
One more thing, why do we use an extra file `labels` if we already have `const`?

Reply:
Normally const and labels can differ. If you want to prepare multi-lang it's easier to do it in a separate file